### ISSUE TITLE
validation data is not true validation data

### ISSUE THREAD

**Issue by harbecke** [harbecke/HexHex#16]
Since validation data is generated every epoch for the training set, it can be seen by the model in previous iterations. Furthermore positions from the same game are very related. We should find a solution to select only a subset of positions from one game or create games only for validation to make sure the validation loss is different from the training loss.
